src/language-html/parser-html.js

```diff
@@ -39,11 +42,54 @@ function ngHtmlParser(
     getHtmlTagDefinition,
   } = require("angular-html-parser/lib/compiler/src/ml_parser/html_tags");
 
-  const { rootNodes, errors } = parser.parse(input, {
+  const parseResult = parser.parse(input, {
     canSelfClose: recognizeSelfClosing,
     allowHtmComponentClosingTags,
     isTagNameCaseSensitive,
+    getTagContentType,
   });
+  const { rootNodes } = parseResult;
+  let { errors } = parseResult;
+
+  if (options.parser === "vue") {
+    const templateNode = rootNodes.find((node) => node.name === "template");
+    const langAttr =
+      templateNode && templateNode.attrs.find((attr) => attr.name === "lang");
+    const langValue = langAttr && langAttr.value;
+    const shouldParseAsHtml =
+      langValue == null || getParserName(langValue, options) === "html";
+
+    if (shouldParseAsHtml) {
+      const secondParseResult = parser.parse(input, {
+        canSelfClose: recognizeSelfClosing,
+        allowHtmComponentClosingTags,
+        isTagNameCaseSensitive,
+      });
```

Comment:
@sosukesuzuki 

Is this secondParse still need with attrs info? I'm okay with this, just double checking with you.

If it's still needed, I think we can optimalize this a little bit, because it's not alway needed, so we can do the second parse when it's needed. Something like 

```js

const getSecondFormat = () => secondResult ? secondResult : (secondResult = doSecondFormat())

if (isUnclosedNode) {
  rootNodes[i] = getSecondFormat().rootNodes
  // ...
} else if (node.name === "template" || node.name === "html") {
  // handle getSecondFormat().errors
}
```

Better?

Reply:
What means "second format"? is not "second parse"?

Reply:
Second parse, sorry, it's not always needed, right?

Reply:
Yes, I'll fix:+1: